kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/extradata/reportingdata/State.java

```diff
@@ -18,6 +18,33 @@ public State(String stateType) {
     this.stateType = stateType;
   }
 
-  
+  /**
+   * @return the priority order if it's considered as a validation state, null otherwise
+   */
+  private Integer getPriorityOrder(){
+    return switch (stateType){
+      case "PARTIELINT" -> 3;
+      case "VALPAP" -> 2;
+      case "VALINT" -> 1;
+      default -> null;
+    };
+  }
+
+  /**
+   * @return true if it's considered as a validation state
+   */
+  public boolean isValidationState(){
+    return (getPriorityOrder() != null);
+  }
+
+  public boolean isPriorityTo(State otherState){
+    if(otherState == null)
+      return true;
+
+    return (
+            this.getPriorityOrder() < otherState.getPriorityOrder()
+            || (this.getPriorityOrder().equals(otherState.getPriorityOrder()) && this.getTimestamp() > otherState.getTimestamp())
```

Comment:
Add a comment on method to explain rules : 
- validation state
- last operation